kubernetes-mixin/dashboards/openai-monitoring.json

```diff
@@ -1,1220 +1,1294 @@
-                "tsNs": true,
-                "type": true
-              },
-              "indexByName": {
-                "Line": 3,
-                "Time": 1,
-                "completion_tokens": 7,
-                "finish_reason": 8,
-                "id": 5,
-                "integration": 9,
-                "labels": 0,
-                "prompt": 2,
-                "prompt_tokens": 10,
-                "total_tokens": 11,
-                "traceID": 6,
-                "tsNs": 4,
-                "type": 12
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
               },
-              "renameByName": {
-                "Line": "Response"
+              {
+                "color": "red",
+                "value": 80
               }
-            }
-          }
-        ],
-        "transparent": true,
-        "type": "table"
+            ]
+          },
+          "unit": "none"
+        },
+        "overrides": []
       },
-      {
-        "datasource": {
-          "type": "prometheus",
-          "uid": "grafanacloud-prom"
+      "gridPos": {
+        "h": 8,
+        "w": 12,
+        "x": 0,
+        "y": 31
+      },
+      "id": 15,
+      "options": {
+        "legend": {
+          "calcs": [],
+          "displayMode": "list",
+          "placement": "bottom",
+          "showLegend": true
+        },
+        "tooltip": {
+          "mode": "single",
+          "sort": "none"
+        }
+      },
+      "pluginVersion": "10.2.0-59542pre",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "$datasource"
+          },
+          "disableTextWrap": false,
+          "editorMode": "code",
+          "exemplar": false,
+          "expr": "openai_requestDuration{job=~\"$job\"}",
+          "format": "table",
+          "fullMetaSearch": false,
+          "includeNullMetadata": true,
+          "instant": false,
+          "legendFormat": "Request Duration",
+          "range": true,
+          "refId": "A",
+          "useBackend": false
         },
-        "fieldConfig": {
-          "defaults": {
-            "color": {
-              "mode": "palette-classic"
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "$datasource"
+          },
+          "disableTextWrap": false,
+          "editorMode": "code",
+          "exemplar": false,
+          "expr": "openai_totalTokens{job=~\"$job\"}",
+          "format": "table",
+          "fullMetaSearch": false,
+          "hide": false,
+          "includeNullMetadata": true,
+          "instant": false,
+          "legendFormat": "Total Tokens",
+          "range": true,
+          "refId": "B",
+          "useBackend": false
+        }
+      ],
+      "title": "Total Tokens vs Request Duration",
```

Comment:
Does it make sense to have both of these on the same timeseries? Request duration is likely to be less than 100 seconds, while average total tokens per request may be over 1000. A 10x scale difference will be hard to grok on the same graph